src/Apps/Order/Routes/Shipping2/Shipping2.jest.tsx

```diff
@@ -913,24 +949,37 @@ describe("Shipping", () => {
               relayEnv
             )
 
-            await fillAddressForm(validAddress)
+            await fillAddressForm(inputAddress)
             userEvent.selectOptions(screen.getByTestId("AddressForm_country"), [
               "TW",
             ])
             await userEvent.click(screen.getByText("Save and Continue"))
 
-            const mutation = env.mock.getMostRecentOperation()
-            expect(mutation.request.node.operation.name).toEqual(
-              "AddressVerificationFlowQuery"
+            await verifyAddressWithSuggestions(env, inputAddress, {
+              ...inputAddress,
+              addressLine1: "<recommended change>",
+            })
```

Comment:
As mentioned above this might be enough to not need to supply a recommended address to this helper.